### ISSUE TITLE
SetIgnoreCaseOnKeyLookup does not work 

### ISSUE THREAD

**Issue by Meowzz95** [StubbleOrg/Stubble#120]
```
using Stubble.Core.Builders;

var renderer = new StubbleBuilder().Configure(builder =>
{
    builder.SetIgnoreCaseOnKeyLookup(true);
}).Build();

var d = new Dictionary<string, string>()
{
    { "someKey", "hello" },
    { "SomeOtherKey", "world" }
};

var template = "{{SomeKey}} {{SomeOtherKey}}";
var res = renderer.Render(template, d);
Console.Out.WriteLine(res);
```

The above sample code prints 
`  world`

Expected:
`hello world`

Version
`<PackageReference Include="Stubble.Core" Version="1.9.3" />`

**Comment by stan-sz** [comment]
Related to #114

**Comment by Romanx** [comment]
Hey @Meowzz95, @stan-sz and @senneclaeskens 

Thanks for your patience this has now been released to nuget as v1.10.8. Thanks for using stubble and letting us know about this
